src/mixins/filing-template-mixin.ts

```diff
@@ -354,7 +354,7 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
     const filing: ContinuationInFilingIF = {
       header: {
         name: FilingTypes.CONTINUATION_IN,
-        certifiedBy: this.getCertifyState.certifiedBy,
+        certifiedBy: this.getCertifyState.certifiedBy || undefined, // remove for authorization
```

Comment:
Without this, we'd store "null" in the JSON, which is not accepted. When the property is undefined, the JS object->JSON mapping removes the property.